json/src/test_helpers/blockchain/mod.rs

```diff
@@ -61,7 +61,7 @@ pub struct BlockChain {
 	/// Blocks.
 	pub blocks: Vec<Block>,
 	/// Post state.
-	pub post_state: State,
+	pub post_state: Option<State>,
```

Comment:
Do you think we should document when it's expected and ok for a test to not have a `post_state`?

Reply:
There is not something that we decide since there are tests that have poststate and other than does not have it, so is up to the ethereum/test team to decide it. In geth this field is also optional.

Reply:
I guess mine was more of a question: what does it mean when a test doesn't have a post-state? Seems like the chain always has a state and I was curious about what kind of test would not have one. :)

Reply:
Oh, yes, if I recall correctly, there is one test (or two there that does not contain postState test). I expected to be an error, but when I see that in geth is optional, I figure out that is safe if we do the same.